assets/queries/cloudFormation/ec2_network_ACL_deny_rules/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "ec2_network_ACL_denied_traffic",
+  "queryName": "EC2 network ACL denied traffic",
+  "severity": "MEDIUM",
+  "category": null,
+  "descriptionText": "The deny rules defined for the selected AWS Network ACL are declared ineffective",
```

Comment:
`"descriptionText": "Ineffective deny rules. A deny rule should be applied to all IP addresses.",`